pkg/daemon/discover/discover.go

```diff
@@ -372,9 +403,51 @@ func probeDevices(context *clusterd.Context) ([]sys.LocalDisk, error) {
 		device.Filesystem = fs
 		device.Empty = clusterd.GetDeviceEmpty(device)
 
+		// Add the information provided by ceph-volume inventory
+		if cvInventory != nil {
+			CVData, device_exists := (*cvInventory)[path.Join("/dev/", device.Name)]
+			if device_exists {
+				device.CephVolumeData = CVData
+			} else {
+				logger.Errorf("ceph-volume information for device %s not found", device.Name)
+			}
+		} else {
+			device.CephVolumeData = ""
+		}
+
 		devices = append(devices, *device)
 	}
 
 	logger.Infof("available devices: %+v", devices)
 	return devices, nil
 }
+
+func GetCephVolumeInventory(context *clusterd.Context) (*map[string]string, error) {
+	inventory, err := context.Executor.ExecuteCommandWithOutput(false, "", "ceph-volume", "inventory", "--format", "json")
+	if err != nil {
+		return nil, fmt.Errorf("failed to execute ceph-volume inventory: %+v", err)
```

Comment:
`		return nil, fmt.Errorf("failed to execute ceph-volume inventory. %+v", err)`

Reply:
Sorry @leseb, I do not understand well your comment here. can you explain ?

Reply:
:-D